statsmodels/compat/numpy.py

```diff
@@ -43,6 +43,8 @@
 from .scipy import NumpyVersion
 import numpy as np
 
+from .python import PY3
```

Comment:
Should group imports here -- two from sm, one foreign.

Reply:
amended and force pushed